python/hail/dataset.py

```diff
@@ -133,7 +133,7 @@ def is_dosage(self):
 
     @handle_py4j
     def file_version(self):
-        """File version of dataset.
+        """File version of VariantDataset.
```

Comment:
We tend to use HailContext, VariantDataset, KeyTable, etc in the top-line method doc, with Hail context, variant dataset, and key table below (except when the class is a link). I'm fine changing all to the later. Thoughts?